smaug/python/graph.py

```diff
@@ -106,6 +113,12 @@ def get_nodes(self):
     """Return nodes in the graph proto."""
     return self.graph.nodes
 
+  def find_data_op_output(self, tensor_name):
+    """Return the output of the data op if one is created for this tensor."""
+    if tensor_name in self._tensor_data_op_map:
+      return self._tensor_data_op_map[tensor_name]
```

Comment:
I believe you can just do `self._tensor_data_op_map.get(tensor_name, None)`.

Reply:
Great advice! This was bugging me.